cmd/validator/flags/flags.go

```diff
@@ -40,6 +40,12 @@ var (
 		Usage: "Beacon node RPC gateway provider endpoint",
 		Value: "127.0.0.1:3500",
 	}
+	// BeaconRESTApiProviderFlag defines a beacon node REST API endpoint.
+	BeaconRESTApiProviderFlag = &cli.StringFlag{
+		Name:  "beacon-rest-api-provider",
+		Usage: "Beacon node REST API provider endpoint",
+		Value: "http://127.0.0.1:3500",
```

Comment:
The prysm beacon node exposes the Beacon REST API on `127.0.0.1:3500` by default though, no? Using `3500` here makes it "just work" for now without having to change anything else. Maybe this can be changed for `4000` later on if/when we completely replace the gRPC endpoints?

Reply:
Yes, it should be 3500, you are right.